docs/research.md

```diff
@@ -0,0 +1,33 @@
+# Research & Discovery
+
+## Existing similar tools and benefits to our project
+
+### 0xFableOrg/roll-op
+- Rollup customization with config file
+
+
+### upnodedev/opstack-compose
+- Dev ops solution (docker-compose + aws)
+- Upnode focuses on blockchain infrastructure (more value)
+- Has a block explorer
+
+## Decisions and drivers
+
+### Developed on Rust
+- Fast, lightweight and compiled
+- Reduces computing costs
+- Increasing popularity
+ 
+## Strategy
+- Core isolated from CLI and App packages enables for Core functionalities testing.
```

Comment:
Also I think to allow for extension and implementation of new multiple clients, in our case the server for the UI, and potentially more